.changeset/four-melons-double.md

```diff
@@ -0,0 +1,6 @@
+---
+'@aws-amplify/ui-react': major
+'@aws-amplify/ui': major
+---
+
+Added new UI primtives and theming API
```

Comment:
@dbanksdesign Can you add some content here that will headline our changelog?

I basically restated the goals & linked the RFC + docs:

https://github.com/aws-amplify/amplify-ui/pull/745/files